### PR TITLE
Add more comments to help implement commands

### PR DESCRIPTION
**Pull request by tmf97**
### Summary

This PR adds more in-line documentation to help implement more commands in the future

### Testing

N/A, just comments

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

**Review by RohitValiveti — APPROVED** [review]
Looks good! Feels nice to be able to understand your comments now 😂.. If you think it would be of use, we could also add documentation for how to write a command test. Your explanations definitely helped me, so just an idea.

**Comment by tmf97** [comment]
Good idea! Will get on how to write a command test

File: utils/constants.py [hunk 1]
```diff
@@ -248,13 +251,12 @@ class DownlinkKwargs(StringEnum):
 ANTENNAE_BURNWIRE_DURATION = 1  # second
 BURN_WAIT_TIME = 1  # minutes changed to glow wait time in params
 
+GOM_TIMING_FUDGE_FACTOR = 3  # milliseconds
```
Comment (sjz38) [inline utils/constants.py:254]: Would help to specify why this is a fudge factor/why we need it

File: communications/commands.py [hunk 2]
```diff
@@ -18,22 +18,35 @@ class Command(ABC):
     All you need to do if you want to implement a new command is to:
```
Comment (sjz38) [inline communications/commands.py:18]: Directions a well written, easy to follow in conjunction with the commands already written

File: communications/commands.py [hunk 3]
```diff
@@ -18,22 +18,35 @@ class Command(ABC):
...
         2. Override the `self._method` method which can only take in a kwarg dictionary: no args other than `parent`
-        3. Add the relevant Codecs - make sure you're unpacking and packing the dict correctly in self._method
-        4. Specify the command's `id`. You'll need to add the id to the CommandEnum in utils.constants
+        3. Add the relevant codec for every variable that you want to be able to uplink and downlink in the class's
+            `uplink_codecs` and `downlink_codecs` lists. Each codec is defined of the name of the variable you want to
+            send, and the data type of that variable. (If you don't have any data to up or downlink, don't worry about
+            this step.)
+        3a. Make sure that every value you define in the uplink_codecs list get unpacked from the `kwargs` dict in the
+            `_method`
+        3b. Make syre that every value you define in the downlink_codecs list get returned in a dict by the `_method`.
```
Comment (sjz38) [inline communications/commands.py:27]: Typo "syre" -> "sure"

**Review by sjz38 — CHANGES_REQUESTED** [review]
👍I'm good to approve, just a few nitpicks

**Comment by sjz38** [comment]
And resolve the conflicting file

File: utils/constants.py (hunk 1 above)
Comment (tmf97) [reply utils/constants.py:254]: Honestly I've forgotten why I put it there. Too scared to delete now 😬

Comment (sjz38) [reply utils/constants.py:254]: Its in attitude_adjustment.py, line 106. Looks like you get rid of it in your timing branch

**Review by sjz38 — APPROVED** [review]
👍

**Pull request merged**